src/tree/CosmosDBAccountWizard/CosmosDBAccountNameStep.ts

```diff
@@ -40,7 +40,7 @@ async function validateCosmosDBAccountName(name: string, client: CosmosDBManagem
         return `The name must be between ${min} and ${max} characters.`;
     } else if (name.match(/[^a-z0-9-]/)) {
         return "The name can only contain lowercase letters, numbers, and the '-' character.";
-    } else if (await client.databaseAccounts.checkNameExists(name)) {
+    } else if ((await client.databaseAccounts.checkNameExists(name)).body.valueOf()) {
```

Comment:
Could you explain some more about what the problem was and how this fixes it? I don't have much context on it

Reply:
With recent package update to "@azure/cosmos", `checkNameExists()` returns a `DatabaseAccountsCheckNameExistsResponse` instead of a boolean.

Reply:
Do you have to use `valueOf`? It looks like `body` is already type boolean

<img width="339" alt="Screen Shot 2020-11-09 at 12 55 06 PM" src="https://user-images.githubusercontent.com/22795803/98595562-fcf9e980-228a-11eb-83ca-cf683c96137e.png">

Reply:
I might have missed it. That's true, valueOf is not required there.